lib/rails_erd/domain.rb

```diff
@@ -120,7 +120,32 @@ def specializations_mapping
     end
 
     def models
-      @models ||= @source_models.select { |model| check_model_validity(model) }.reject { |model| check_habtm_model(model) }
+      @models ||= @source_models
+                    .reject { |model| tableless_rails_models.include?(model) }
+                    .select { |model| check_model_validity(model) }
+                    .reject { |model| check_habtm_model(model) }
+    end
+
+    # Returns Rails model classes defined in the app
+    def rails_models
+      %w(
+        ActionMailbox::InboundEmail
+        ActiveStorage::Attachment
+        ActiveStorage::Blob
+        ActiveStorage::VariantRecord
+        ActionText::RichText
+        ActionText::EncryptedRichText
+      ).map{ |model| Object.const_get(model) rescue nil }.compact
+    end
+
+    def tableless_rails_models
+      @tableless_rails_models ||= begin
```

Comment:
The `begin` here is catching the `rescue` from L23 above, yes? Can we move it there so they're clearly tied together? WDYT?

Reply:
🤔 No it is memoizing the call to check if the table exists (if `Rails`) 

```
          rails_models.reject{ |model| model.table_exists? }
```

Reply:
AH! I  see.. got it :)